.github/scripts/workflow_rerun/rerunner.py

```diff
@@ -59,8 +59,8 @@
         # Needed to run a step after for statistics
         with open(file=os.environ['GITHUB_ENV'],
                   mode='a') as fh:
-            fh.write('PIPELINE_RETRIGGERED=true')
-            fh.write(f'FOUND_ERROR_TICKET={log_analyzer.found_error_ticket}')
+            fh.write('PIPELINE_RETRIGGERED=true\n')
```

Comment:
W/o the newline the `GITHUB_ENV` file looked like `PIPELINE_RETRIGGERED=trueFOUND_ERROR_TICKET=123456` which was not parsed correctly.